.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,40 +7,97 @@ assignees: ''
 
 ---
 
-**Describe the bug**
+**NB** TopoStats `< 2.0` is no longer actively maintained. If you have encountered an issue using this deprecated
+version please try TopoStats `>= 2.0.0`, there are instructions on
+[installation](https://afm-spm.github.io/TopoStats/installation.html),
+[usage](https://afm-spm.github.io/TopoStats/usage.html) and
+[configuration](https://afm-spm.github.io/TopoStats/configuration.html).
+
+## Checklist
+
+Please try and tick off each of these items when filing the bug report. There are further instructions on each below.
+
+* [ ] Describe the bug.
+* [ ] Include the configuration file.
+* [ ] Copy of the output.
+* [ ] The exact command that failed. This is what you typed at the command line, including any options.
+* [ ] TopoStats version, this is reported by `run_topostats --version`
+* [ ] Operating System and Python Version
+
+## Describe the bug
 A clear and concise description of what the bug is.
 
-**To Reproduce**
+## Copy of the output
+
+Please copy and paste the output that is shown below within the `\`` (triple-backticks).
+
+```
+<-- PASTE OUTPUT HERE -->
+```
+
+
+## Include the configuration file.
+
+If no configuration file was specified with the `-c`/`--config-file` option the defaults were used, please use the
+`run_topostats --create-config-file crash.yaml` to save these to the `crash.yaml` file and copy the contents below.
+
+``` yaml
+<-- PASTE CONTENT OF crash.yaml HERE -->
+```
 
-- Command used to run topostats e.g. `python pygwytracing.py` / `python topostats/topotracing.py`
-- Config file (you can attach this or paste it below)
 
-**Expected behavior**
-A clear and concise description of what you expected to happen.
+## To Reproduce
 
-**Output**
+If it is possible to share the file that caused the error that would greatly assist in reproducing and investigating the problem. In addition the _exact_ command used that failed should be pasted below.
```

Comment:
Would it be worth explaining to the user what to do (eg google drive link) if the file cannot be shared publicly? Also, `.spm` files aren't supported by GitHub, so this should be explained too